client/src/components/ReviewTile.js

```diff
@@ -1,11 +1,15 @@
 import React from "react";
 
-const ReviewTile = ({ title, rating, content }) => {
+const ReviewTile = ({ id, title, rating, content, deleteReview }) => {
+
   return (
     <div>
       <h4>{title}</h4>
       <h5>{rating}</h5>
       <p>{content}</p>
+      <div onClick={() => { deleteReview(id) }}>
+        <input type="submit" value="Delete Review"/>
```

Comment:
you might also consider only making this delete visible if the signed in user is the same one who created the review